web/src/components/organisms/Project/Content/ContentList/index.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from "@emotion/styled";
+import { useParams, useNavigate } from "react-router-dom";
+
+import Content from "@reearth-cms/components/atoms/Content";
+import ContentTable from "@reearth-cms/components/molecules/Content/ContentTable";
+import ModelsMenu from "@reearth-cms/components/organisms/Project/ModelsMenu";
+import { useT } from "@reearth-cms/i18n";
+
+const ContentList: React.FC = () => {
+  const t = useT();
+
+  const navigate = useNavigate();
+
+  const { projectId, workspaceId } = useParams();
+  const selectModel = (modelId: string) => {
+    navigate(`/workspaces/${workspaceId}/${projectId}/content/${modelId}`);
+  };
+
+  return (
+    <PaddedContent>
+      <SchemaStyledMenu>
+        <ModelsMenu title={t("Content")} selectModel={selectModel} />
+      </SchemaStyledMenu>
```

Comment:
The naming here is confusing. In your refactor please be careful to update ALL names that use to reference Schema but now should be Menu (or whatever else is most appropriate). 
It looks like SchemaStyledMenu is just a wrapper, so either rename it or just make a StyledModelsMenu and add the width directly to it.

Reply:
https://github.com/reearth/reearth-cms/blob/519f18972394b845c74963b3b1b4f68b5c2b54bd/web/src/components/organisms/Project/Content/ContentList/index.tsx#L25-L27